### PR TITLE
use northstar

### LINKED ISSUE #16: Add Northstar integration & RBAC
If we want to deploy as a public app, we'll need the same level of auth as for, e.g., Guardian.

### PR DESCRIPTION
**Pull request by katiecrane**
- do lots of things [similar to what we did](https://github.com/DoSomething/voting-app/pull/514) in Jubilee 🎉  for Northstar-PHP
- previously we had no auth at all #lawless
- add login view
- add login/logout to the top navigation bar
- we have admin middleware now! users have to be logged in AND admins to get to anything except the homepage (which reveals 0 information)
- #puddlegram
### Issues:

Fixes #16

### CODE REVIEW

File: resources/views/auth/emails/password.blade.php [hunk 1]
```diff
@@ -0,0 +1 @@
+Click here to reset your password: <a href="{{ $link = url('password/reset', $token).'?email='.urlencode($user->getEmailForPasswordReset()) }}"> {{ $link }} </a>
```
Comment (DFurnes) [inline resources/views/auth/emails/password.blade.php]: We aren't using this, right?

Comment (katiecrane) [inline resources/views/auth/emails/password.blade.php]: We are not! I'll take another clean-up pass. Good 👀

**Comment by DFurnes** [comment]
Also looks like there are some StyleCI buggers! 🎭 And failing tests! 🚥 💥 

Otherwise looks pretty good to my 👀.

**Comment by katiecrane** [comment]
@DFurnes Alright I'm on clean-up and StyleCI fun times. 

I talked to @blisteringherb about wercker and he said it's not getting automatically pulled in to build so what it's saying now is expected.

**Comment by DFurnes** [comment]
Oh, that's a bummer. 😞

**Comment by katiecrane** [comment]
@DFurnes can you 👀  the new commits? Nothing too wild

**Comment by DFurnes** [comment]
🚢 🌠

**Pull request merged**